build/publish/Microsoft.DotNet.Cli.Checksum.targets

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="GenerateChecksums"
+          DependsOnTargets="Init;
+                            SetupGenerateChecksumsInputs;"
+          Inputs="@(GenerateChecksumsInputs)"
+          Outputs="%(GenerateChecksumsInputs.DestinationPath)">
+    <GenerateChecksums Items="@(GenerateChecksumsInputs)" />
```

Comment:
nit: 
`GenerateChecksumsInputs` doesn't tell me much about the contents of these inputs. 

Can you rename this property to reflect what type of inputs it is?